libkbfs/folder_block_ops.go

```diff
@@ -36,6 +36,9 @@ const (
 	blockRead blockReqType = iota
 	// A block write request.
 	blockWrite
+	// A block read request that is happening from a different
+	// goroutine than the blockLock rlock holder, using the same lState.
```

Comment:
Ok, I added panics for a nil lState in that case, and made the `folderBlockOps` `fileData` pass that in for parallel block reads in its getter function.